pkgs/tools/networking/modemmanager/default.nix

```diff
@@ -89,14 +89,18 @@ stdenv.mkDerivation rec {
   # In Nixpkgs g-ir-scanner is patched to produce absolute paths, and
   # that interferes with ModemManager's tests, causing them to try to
   # load libraries from the install path, which doesn't usually exist
-  # when `make check' is run.  So to work around that, we run it as an
+  # when `meson test' is run.  So to work around that, we run it as an
   # install check instead, when those paths will have been created.
   doInstallCheck = true;
-  preInstallCheck = ''
+  installCheckPhase = ''
+    runHook preInstallCheck
     export G_TEST_DBUS_DAEMON="${dbus}/bin/dbus-daemon"
     patchShebangs tools/tests/test-wrapper.sh
+    substituteInPlace tools/tests/test-wrapper.sh \
+      --replace-fail GI_TYPELIB_PATH= GI_TYPELIB_PATH=$GI_TYPELIB_PATH:
```

Comment:
Maybe quote this just to be safe. Ideally, we would remove this upstream (example in https://gitlab.gnome.org/GNOME/libsecret/-/merge_requests/141)

Reply:
Or at least add a comment that this is to find GLib.

Reply:
I was thinking about that, but I think it's actually safer _not_ to quote, because it's less likely that Nix paths will start to include spaces than upstream will add quotes and break our extra quotes inside.

Reply:
Good point, looks like it is not space safe upstream:

https://gitlab.freedesktop.org/mobile-broadband/ModemManager/-/blob/218d6d2c5879e97d5f564aff8fa2546212decdfa/tools/tests/test-wrapper.sh.in#L5

Reply:
Thanks for the example.  I've opened https://gitlab.freedesktop.org/mobile-broadband/ModemManager/-/merge_requests/1179.

Reply:
Okay, updated to use the now merged upstream patch.